frontend/src/Signin.js

```diff
@@ -0,0 +1,59 @@
+import styles from './auth.module.css';
+import classNames from 'classnames';
+import axios from 'axios';
+import {useState} from 'react';
+
+function SignIn() {
+  const [userEmail, setUserEmail] = useState (''); 
+  const [userPassword, setUserPassword] = useState ('');
+
+  const signIn = async (_event) => { // called when LOGIN button is clicked
+    const response = await axios.post('/sign-in', {
+      email: userEmail,
+      password: userPassword,
+    }); 
+    console.log("sign in was successful for ", response.data)
+  }
+  const emailHandler = (event) => { 
```

Comment:
similarly, this might be clearer as "handleEmailChange"

let's do a pass on all of the handler functions below as well